lib/shippo.rb

```diff
@@ -1,4 +1,4 @@
-#
+VERSION = '2.0.5'#
```

Comment:
@soloncyang This is unnecessary – the version is currently stored in a separate file `lib/shippo/api/version.rb`. 

If you want to up the version, please change it in that file.

https://github.com/goshippo/shippo-ruby-client/blob/master/lib/shippo/api/version.rb